gridt/src/app/login/register/register.page.spec.ts

```diff
@@ -11,7 +11,15 @@ describe("RegisterPage", () => {
   //testing suite, (de pagina te testen)
```

Comment:
Can you remove my comments? They don't fit and should have been removed by me before pushing.